app/src/main/kotlin/it/vfsfitvnm/vimusic/Database.kt

```diff
@@ -198,6 +198,9 @@ interface Database {
     @Query("SELECT * FROM Format WHERE songId = :songId")
     fun format(songId: String): Flow<Format>
 
+    @Query("SELECT * FROM Song JOIN Format ON id = songId WHERE isDownloaded = true ORDER BY Song.ROWID DESC")
```

Comment:
should it be `true` of `1` ?

Reply:
I guess true, because we mark column isDownloaded as Boolean

Reply:
ok let's try if it works. I'm confused because I saw it is INTEGER in https://github.com/jasper-apps/ViMusic/pull/18/files#diff-96f61b197b16d953532da7e423cc1f9151a523b9b79b6348f098b3efd3aa9795